src/js/orders.spec.ts

```diff
@@ -77,13 +77,14 @@ export class Order {
     };
   }
 
-  asArray(): [string, string, string, string, string] {
+  asArray(): [string, string, string, string, string, string] {
```

Comment:
ubernit: Not really related to this PR, but I think this should be called `asTuple`, as that is the Solidity ABI type that is being encoded, and the TypeScript type being used (`[string, string]` is a tuple, `string[]` is an array and they are very different from a type perspective).
```suggestion
  asTuple(): [string, string, string, string, string, string] {
```

Reply:
Sure, should I just change it here?

Reply:
I remove this function in another PR because it's unused by the way.